src/aks-preview/azcli_aks_live_test/configs/ext_matrix_default.json

```diff
@@ -24,6 +24,7 @@
             "test_aks_custom_ca_trust_flow",
             "test_aks_create_with_csi_driver_v2",
             "test_aks_create_and_update_csi_driver_to_v2"
+            "test_aks_nodepool_abort"
```

Comment:
missing a trailing comma on the above line.

Reply:
Added